client/app/scripts/superdesk-templates/templates.js

```diff
@@ -156,8 +157,37 @@
                 desks.initialize()
                 .then(function() {
                     $scope.desks = desks.desks;
+
+                    /*
+                     * Returns desk name
+                     */
+                    $scope.getTemplateDesk = function getTemplateDesk(template) {
+                        return _.find($scope.desks._items , {_id: template.template_desk});
+                    };
+
+                    /*
+                     * Returns stage name
+                     */
+                    $scope.getTemplateStage = function getTemplateStage(template) {
+                        return _.find(desks.stages._items , {_id: template.template_stage});
+                    };
                 });
 
+                /*
+                 * Convert string to time object
+                 *
+                 * @param {String} time
```

Comment:
I recommend describing the expected format of the input string, it is probably not apparent that it is `%M%S` (?).